examples/fluids/src/smartsim/sgs_dd_training.c

```diff
@@ -231,14 +231,18 @@ PetscErrorCode SGS_DD_TrainingSetup(Ceed ceed, User user, CeedData ceed_data, Pr
       array_info[4] = smartsim->collocated_database_num_ranks;
       array_info[5] = rank;
 
-      PetscSmartRedisCall(put_tensor(smartsim->client, "sizeInfo", 8, array_info, &array_info_dim, 1, SRTensorTypeInt32, SRMemLayoutContiguous));
+      PetscCall(PetscLogEventBegin(SmartRedis_Meta, 0, 0, 0, 0));
+      PetscSmartRedisCall(put_tensor(smartsim->client, "sizeInfo", 8, array_info, &array_info_dim, 1, SRTensorTypeInt64, SRMemLayoutContiguous));
```

Comment:
Any reason for changing to the `SRTensorTypeInt32` to 64bit?

Reply:
I remember making this change because the SR API on the Python side assumes a 64 bit Int and this was causing some issues, but I'll double check.

Reply:
Yeah, we end up reading garbage for these values if it's not a 64 bit Int. I think we should keep it for now and I can investigate why later.

Reply:
Hmm. Ok. That might cause it to fail in the test harness here, but we can see.

Reply:
Reverted to 32 bit Int, checked that training reads correct data